problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,60 @@
+const MyQueue = function () {
+  this.stack1 = [];
+  this.stack2 = [];
+};
+
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.stack1.push(x);
+};
+
+/**
+ * leetcode problem link: https://leetcode.com/problems/implement-queue-using-stacks
+ *
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  if (this.stack2.length) {
+    return this.stack2.pop();
+  }
+
+  while (this.stack1.length) {
+    this.stack2.push(this.stack1.pop());
+  }
+
+  return this.stack2.pop();
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.peek = function () {
+  if (this.stack2.length) {
+    return this.stack2.at(-1);
+  }
+
+  while (this.stack1.length) {
+    this.stack2.push(this.stack1.pop());
+  }
+
+  return this.stack2.at(-1);
+};
```

Comment:
```suggestion
  if (this.stack2.length) {
    return this.stack2.at(-1);
  } else {
    return this.stack1[0];
  }
};
```

stack2로 stack1 데이터를 옮기는 작업은 pop 메서드 안에서만 이뤄지고 peek 메서드에서는 
while문 없이 이뤄진다면 시간복잡도를 좀 더 개선할 수 있을 것 같아요.

Reply:
오 시간복잡도 줄이기 좋네요! 감사합니다.